README.md

```diff
@@ -69,7 +69,7 @@ After the binary downloads, please move it to a location in your shell's `$PATH`
 Let's build a solution for a very common topology. A application service and an application client running
 in two private networks wish to communicate with each other without exposing ports on the Internet.
 
-```bash
+``bash
```

Comment:
The triple backticks introduce a code block (see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) and AFAICS the syntax was correct.

Reply:
Sorry sir actually I forget it & can I resolve.